app/views/shared/_application_information.html.erb

```diff
@@ -66,6 +66,42 @@
       </table>
     </div>
   </div>
+
+  <div class="govuk-accordion__section">
+    <div class="govuk-accordion__section-header">
+      <h3 class="govuk-accordion__section-heading">
+        <!-- We have removed aria controls due to a problem with the default npm gov-uk functionality, which forces accordions to stay open by using the aria controls classes to setState in the browser, and force them open.
+        Our research suggests this makes the website harder to use as there is too much information on the screen overwhelming the user.
+        For now we are removing these until we create our own version of the gov-uk library that stops the behaviour of forcing accordions open. -->
+        <button type="button" id="accordion-default-heading-2" aria-controls="" class="govuk-accordion__section-button" aria-expanded="false">
+          Site Map
+        </button>
+          <span class="govuk-accordion__icon" aria-hidden="true"></span>
+      </h3>
+    </div>
+    <div class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
+      <% if @planning_application.boundary_geojson.present? %>
+        <div id='map' style="width: 100%; height: 200px"></div>
+        <script>
+          var map = L.map('map', {minZoom: 8, maxZoom: 22});
+          L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+            maxZoom: 22,
+            maxNativeZoom: 19
+          }).addTo(map);
+          var boundary = L.geoJSON(<%= JSON.parse(@planning_application.boundary_geojson).to_json.html_safe %>, {"color": "#ff0000", "weight": 2, "opacity": 0.65}).addTo(map);
+          map.fitBounds(boundary.getBounds());
+
+          function accordion_expanded() {
+            map.invalidateSize().fitBounds(boundary.getBounds());
+          }
+        </script>
```

Comment:
curious about us using JS within the erb as it breaks convention with the rest of the app - is this what was necessary to override the native JS library? otherwise we should keep it separate